ocaml/xapi/xapi_cluster.ml

```diff
@@ -40,10 +40,10 @@ let create ~__context ~cluster_stack ~pool_auto_join ~token_timeout ~token_timeo
       let cluster_host_uuid = Uuidm.to_string (Uuidm.create `V4) in
       (* For now we assume we have only one pool
          TODO: get master ref explicitly passed in as parameter*)
-      let pool = Db.Pool.get_all ~__context |> List.hd in
-      let host = Db.Pool.get_master ~__context ~self:pool in
+      let host = Helpers.get_master ~__context in
+      let network = Db.Network.get_all ~__context |> List.hd in
```

Comment:
This looks wrong and I think is reverted by a subsequent commit. It would be better to squash this commit with th e other one that removes the network parameter from everywhere, then the final diff of each commit would make sense.